backend/endpoints/rooms.py

```diff
@@ -61,3 +63,69 @@ async def create_room(
         status_code=status.HTTP_424_FAILED_DEPENDENCY,
         detail="unable to create room",
     )
+
+
+@router.put(
+    "/org/{org_id}/rooms/{room_id}/members/{member_id}",
+    status_code=status.HTTP_200_OK,
+    responses={
+        400: {"detail": "the max number for a Group_DM is 9"},
+        401: {"detail": "member not an admin"},
+        403: {"detail": "DM room or not found"},
+    },
+)
+async def add_to_room(data: AddToRoom, org_id: str, room_id: str, member_id: str):
+    """Adds a new member(s) to a room
+    Args:
+        data: a pydantic schema that defines the request params
+        org_id (str): A unique identifier of an organisation
+        room_id: A unique identifier of the room to be updated
+
+    Returns:
+        HTTP_200_OK: member added
+    Raises:
+        HTTP_400_BAD_REQUEST: the max number for a Group_DM is 9
+        HTTP_401_UNAUTHORIZED: member not an admin
+        HTTP_403_FORBIDDEN: DM room or not found
+    """
+
+    new_member = data.dict().get("new_member")
```

Comment:
Good work.
What if we created a add_to_room function in the room_utils where most of this codes are placed in and also this would allow the function to be used any where, so we therefore reference the function inside this endpoint ?

Reply:
That sounds great, but I don't see this function reusable anywhere else in the code base. The purpose is unilateral and all possible use cases have been factored in.

Reply:
Yeah, I have not seen other general purpose for it too though.
I just felt separating the actual business logic code from the controller logic, might help in re-usability.
But also this is also is not consequential for now as the purpose is unilateral.